ecl/eclagent/eclagent.cpp

```diff
@@ -3629,7 +3629,7 @@ extern int HTHOR_API eclagent_main(int argc, const char *argv[], StringBuffer *
                 }
 
                 const char * appName = argv[0];
-                for (int finger=0; argv[0][finger] != (const char)NULL; finger++)
+                for (int finger=0; argv[0][finger] != (char)NULL; finger++)
```

Comment:
Should be '\0' rather than (char) NULL

Reply:
Looks like Jake already fixed that issue in a separate PR...

Reply:
Removed the commit that touches eclagent.cpp since Jake fixed it.